Documentation/ceph-object-bucket-claim-crd.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Object Bucket Claim CRD
+weight: 2800
+indent: true
+---
+
+# Ceph Object Bucket Claim CRD
+
+Rook allows creation and customization of object bucket claim through the custom resource definitions (CRDs). The following settings are available for Ceph Object Bucket Claim. It also requires storage class
```

Comment:
Change to:
Rook supports the creation of new buckets (and access to existing buckets) via two custom resources:
- an Object Bucket Claim (OBC) is custom resource which requests a bucket (new or existing) and is described by a Custom Resource Definition (CRD) shown below.
- an Object Bucket (OB) is a custom resource generated when a bucket is provisioned and is described by an OB CRD, also shown below.

An OBC references a storage class which is created by an administrator. The storage class defines whether the bucket requested is a new bucket or an existing bucket. It also defined the bucket retention policy.

Reply:
The OB and OBC CRDs need to be shown below.

Reply:
Done